api/v1alpha1/hostedcluster_types.go

```diff
@@ -23,6 +23,10 @@ type HostedClusterSpec struct {
 	// workers. It should have an ".dockerconfigjson" key containing the pull secret JSON.
 	PullSecret corev1.LocalObjectReference `json:"pullSecret"`
 
+	SigningKey corev1.LocalObjectReference `json:"signingKey"`
+
+	IssuerURL string `json:"issuerURL"`
```

Comment:
This is API we never really talked about much, do they warrant a new field to encapsulate them? Maybe we should doc them at least

Reply:
Yes, I was thinking that as well.  An `IAMSpec` struct or something.

Reply:
We can do it in a followup